includes/abstracts/abstract-wc-payment-token.php

```diff
@@ -104,7 +107,7 @@ public function get_type( $context = 'view' ) {
 	 * @return string
 	 */
 	public function get_display_name( $context = 'view' ) {
```

Comment:
Yeah, since it's not an editable (?) thing by itself, I don't think it will ever need context.